scripts/set_restart_year.sh

```diff
@@ -1,110 +0,0 @@
-#!/bin/bash
-#  Copyright 2020 Scott Wales
-#
-#  \author  Scott Wales <scott.wales@unimelb.edu.au>
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
-# Sets the start year in each model component
-
-source /etc/profile.d/modules.sh
-module use /g/data/hh5/public/modules
-module load conda/analysis3
-module load nco
-
-set -eu
-trap "echo Error in set_restart_year.sh" ERR
-export UMDIR=~access/umdir
-
-# Load some helper scripts
-SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-source $SCRIPTDIR/utils.sh
-
-# Starting year
-start_year=$1
-
-# Set the restart year in the atmosphere and ice namelists
-set_um_start_year $start_year
```

Comment:
Noticed that these sectionsthe control directory namelists. It looks like they are already set to the right values of 1850-01-01 though so everything looks ok.